### ISSUE TITLE
[A11Y] Keyword list stuck on screen

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9328]
## Issue Description
When adding keywords, if the user press "tab" the cursors moves out from the field and the list of suggested keywords remains. It is not possible to remove it using keyboard only.

## How to reproduce / Current behavior
1. Create a submission
2. Go to the first step "Make a Submission: Details"
3. Add a keyword which already exists, e.g. "water"
4. When the recommend keyword list shows up press "tab" key
5. The list remains on screen and it is not possible to select the keyword using keyboard only

## What application are you using?
 OJS 3.4.0

## Stack used 
- Device: Laptop
- OS: MacOS(Monterey)
- Browser: Chrome
- Version: 117

## Testing method
- Method: Manual - Keyboard

## Action
Pressing the TAB key  should move away from the field and remove the suspended list as well. 
Also, it should be Included a combination of instructions and a "Google recommended search" approach. E.g.: [https://alphagov.github.io/accessible-autocomplete/examples/](https://www.google.com/url?q=https://alphagov.github.io/accessible-autocomplete/examples/&sa=D&source=editors&ust=1695777390985066&usg=AOvVaw3xfsbvWy4Pb6bnQJhcMAcG)

## Additional information

![Image](https://github.com/pkp/pkp-lib/assets/4992902/c880ff60-cec8-4306-a19b-bf33b0b6d4b1)

**Comment by Devika008** [comment]
Building on to the solution proposed and as discussed by Israel and I, what can happen is the approach shown [here](https://alphagov.github.io/accessible-autocomplete/examples/form.html), wherein "TAB" is used to select a field, "UP and DOWN" arrows are used for navigating the auto suggested dropdown, "ENTER" is used to select the option we want to enter and then "TAB" is clicked again to move to the next form field

https://github.com/pkp/pkp-lib/assets/112014397/ec5c8c3a-8778-459c-af63-cdc67af5b103
